modules/core/src/v2/coins/xrp.ts

```diff
@@ -354,9 +355,9 @@ export class Xrp extends BaseCoin {
   public verifyTransaction({ txParams, txPrebuild }: VerifyTransactionOptions, callback): Bluebird<boolean> {
     const self = this;
     return co<boolean>(function *() {
-      const explanation = yield self.explainTransaction({
+      const explanation = (yield self.explainTransaction({
         txHex: txPrebuild.txHex,
-      });
+      })) as any;
```

Comment:
Why did a bunch of things get changed to `as any` here? This is in a bunch of lines above

Reply:
This is due to the typescript version upgrade, specifically [this version](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-6.html#stricter-generators) which changed how generator function return types are inferred. Previously, this would just be `any`, but now it's inferring that the return type can be `undefined` (even though this should be covered by `any`, which I still don't full understand) and that caused a compiler error any time we did a `yield` and then unconditionally used the return value.

Reply:
The return type was already `any` prior to this change, so there shouldn't be any behavior change here whatsoever.

Reply:
ahhh interesting. that's annoying. it's just more explicit now